src/ui.js

```diff
@@ -1,20 +1,38 @@
 var $ = require('jquery');
 
 function UI() {
+  var playerMarker = ""
 }
 
 UI.prototype.displayTurn = function(marker) {
   $("#turn-label").html(marker + "'s turn!");
+  playerMarker = marker;
 }
 
-UI.prototype.listenForSpotClick = function(makeMove) {
-  this.callOnElementClick(".spot", makeMove);
+UI.prototype.listenForSpotClick = function(spotClicked) {
+  this.callOnElementClick(".spot", spotClicked);
 }
 
 UI.prototype.callOnElementClick = function(element, callback) {
   $(element).on("click", function(e) {
-    callback(e);
+    var id = callback(e);
+    UI.prototype.displayMarker(id, playerMarker);
+    playerMarker = UI.prototype.switchMarker(playerMarker);
```

Comment:
The logic of who's turn it is should DEFINITELY not be in the UI, take this to game. (I know you mentioned you were doing this anyways)